docs/reference/ilm/ilm-index-lifecycle.asciidoc

```diff
@@ -36,6 +36,17 @@ the index must be older than the minimum age of the next phase.
 The minimum age defaults to zero, which causes {ilm-init} to move indices to the next phase
 as soon as all actions in the current phase complete. 
 
+If an index has unallocated shards and the cluster is in the _yellow_ state, 
+the index can still transition from one phase to the next according to its {ilm} policy, 
+but there might be unexpected side effects because {es} can only perform 
+certain clean up tasks on a healthy cluster. 
+For example, if you have a policy that moves shards to warm nodes when 
+the index transitions out of the hot phase, {es} won't delete shards that 
+weren't fully replicated from the hot nodes. 
```

Comment:
I took a stab at the suggestion, but feel free to ignore if I'm off base. 
I think this might benefit from being in its own para.
```suggestion

For example, when an index transitions from the hot phase to the warm phase,
your policy moves shards from hot nodes to warm ones. For a yellow cluster,
{es} won't clean up any shards on hot nodes that weren't fully replicated. For
a healthy green cluster, these shards would be deleted as part of cleanup.

```

Reply:
I think the example is just making this more complicated than it needs to be. The key takeaway is that the index will still move through the lifecycle, but bad things can happen.